src/components/common/RecipientsTable/RecipientTable.js

```diff
@@ -408,7 +412,7 @@ const RecipientTable = ({
             <Popconfirm
               title="Sure to delete?"
               onConfirm={() => {
-                deleteRecipient(record.id);
+                deleteRecipient(record.recipient_id);
```

Comment:
Thanks for fixing this. The old "id" column was leftover from the database refactor a few months back.